docs/software/cloud.md

```diff
@@ -962,8 +963,38 @@ Rclone is a command-line program to manage files on cloud storage. It is a featu
 
 ***
 
-Official documentation: <https://rclone.org/>  
-Source code: <https://github.com/rclone/rclone>  
+Official documentation: <https://rclone.org/>
+Source code: <https://github.com/rclone/rclone>
 License: [MIT](https://github.com/rclone/rclone/blob/master/COPYING)
 
+## Restic
+
+Restic is a fast and secure backup program.
+
+![Restic logo](../assets/images/restic-logo.png){: width="300" height="69" loading="lazy"}
+
+=== "Quick start"
+
+In order to create a repository at `/srv/restic-repo`, run the following command and enter the same password twice:
```

Comment:
I suggest to use the DietPi userdata path to align with other software options on DietPi:
```suggestion
In order to create a repository at `/mnt/dietpi_userdata/restic-repo`, run the following command and enter the same password twice:
```